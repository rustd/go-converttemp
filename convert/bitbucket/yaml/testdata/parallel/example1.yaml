pipelines:
  default:
    - step: # non-parallel step
        script:
          - ./build.sh
    - parallel: # these 2 steps will run in parallel
        steps:
          - step:
              script:
                - ./integration-tests.sh --batch 1
          - step:
              script:
                - ./integration-tests.sh --batch 2
    - step: # non-parallel step
        script:
          - ./deploy.sh
