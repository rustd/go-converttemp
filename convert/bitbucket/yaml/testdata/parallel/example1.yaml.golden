{
    "pipelines": {
        "default": [
            {
                "step": {
                    "script": [
                        "./build.sh"
                    ]
                }
            },
            {
                "parallel": {
                    "steps": [
                        {
                            "step": {
                                "script": [
                                    "./integration-tests.sh --batch 1"
                                ]
                            }
                        },
                        {
                            "step": {
                                "script": [
                                    "./integration-tests.sh --batch 2"
                                ]
                            }
                        }
                    ]
                }
            },
            {
                "step": {
                    "script": [
                        "./deploy.sh"
                    ]
                }
            }
        ]
    }
}